src/Sarif.Viewer.VisualStudio.Core/CodeAnalysisResultManager.cs

```diff
@@ -45,12 +45,14 @@ internal sealed class CodeAnalysisResultManager : IVsSolutionEvents
         internal const uint VSCOOKIE_NIL = 0;
         internal const int S_OK = 0;
         private const string AllowedDownloadHostsFileName = "AllowedDownloadHosts.json";
+        private const string AllowedFileExtensionsFileName = "AllowedFileExtensions.json";
         private const string TemporaryFileDirectoryName = "SarifViewer";
         private readonly string temporaryFilePath;
 
         // Cookie for registration and unregistration
         private uint m_solutionEventsCookie;
         private readonly List<string> _allowedDownloadHosts;
+        private readonly List<string> _allowedFileExtensions;
```

Comment:
>List [](http://example.com/codeflow?start=25&length=4)

this could be a hashset

Reply:
changed to hashset .also can have better searching performance.